docs/src/catalyst_applications/bifurcation_diagrams.md

```diff
@@ -1,115 +1,120 @@
+
+To illustrate this, we will create a simple model of a kinase that is produced and degraded (at rates $p$ and $d$). The kinase facilitates the phosphorylation of a protein ($X$), which is dephosphorylated at a constant rate. For this system, we will compute a bifurcation diagram, showing how the concentration of the phosphorylated protein ($Xp$) depends on the degradation rate of the kinase ($d$). We will set the total amount of protein ($X+Xp$) to $1.0$.
+```@example ex2
+using BifurcationKit, Catalyst, Plots
+kinase_model = @reaction_network begin
+    (p, d), 0 <--> K
+    (K*kP,kD), X <--> Xp
+end
+
+u_guess = [:K => 1.0, :X => 1.0, :Xp => 1.0]
+p_start = [:p => 1.0, :d => 0.5, :kP => 2.0, :kD => 5.0]
+u0 = [:X => 1.0, :Xp => 0.0]
+bprob = BifurcationProblem(kinase_model, u_guess, p_start, :d; plot_var=:Xp, u0=u0)
+
+p_span = (0.1, 10.0)
+opts_br = ContinuationPar(p_min = p_span[1], p_max = p_span[2], max_steps = 1000)
+bif_dia = bifurcationdiagram(bprob, PALC(), 2, (args...) -> opts_br; bothside=true)
+plot(bif_dia; xguide="d", yguide="Xp")
 ```
+This bifurcation diagram does not contain any interesting features (such as bifurcation points), and only shows how the steady state concentration of $Xp$ is reduced as $d$ increases. 
 
-Here, the Hopf bifurcation is marked with a red dot and the fold bifurcations
-with blue dots. The region with a thinner line width corresponds to unstable
-steady states.
+Finally, for additional clarity, we reiterate the purpose of the two `u` arguments used:
+- `u_guess`: A guess of the initial steady states (which BifurcationKit uses to find its starting point). Typically, most trivial guesses work (e.g. setting all species concentrations to `1.0`). `u_guess` *does not* have to fulfil the conserved concentrations provided in `u0`.
+- `u0`: Used to compute the concentrations of any conserved quantities (e.g. in our example $X + Xp = 1.0$). Technically, values are only required for species that are involved in conservation laws (in our case we do not need to provide a value for $K$). However, sometimes determining which species are actually involved in conservation laws can be difficult, and it might be easier to simply provide concentrations for all species.
+
+!!! note
+    Computing bifurcation diagrams for [hierarchical models created by composing multiple systems](@ref compositional_modeling), that also contain conservations laws, is currently not supported. For these, please [flatten](@ref api_network_extension_and_modification) the system first.
```

Comment:
I thought we had discussed this. What isn't working here? Conversion to a nonlinear system should already flatten implicitly.

Reply:
fixed